frontend/pkg/frontend/frontend_test.go

```diff
@@ -59,6 +60,12 @@ func newClusterResourceID(t *testing.T) *azcorearm.ResourceID {
 	return resourceID
 }
 
+func newNoopAuditClient(t *testing.T) *audit.AuditClient {
+	c, err := audit.NewOtelAuditClient(0, "", true)
+	require.NoError(t, err)
+	return c
+}
```

Comment:
I won't ask for it in this PR, but we should establish a `testhelpers.go` file with these kinds of functions for frontend unit tests, similar to `/internal/api/testhelpers.go`.

Reply:
I jstarted it with this PR and moved the function out